msa_sdk/repository.py

```diff
@@ -5,6 +5,8 @@
 
 from msa_sdk.msa_api import MSA_API
 
+import json
```

Comment:
Is this import necessary ?

Reply:
yes, it is required,   earlier json is imported at every method.  I moved it to file level